db/migrate/20150128185746_seed_new_storytime_actions_and_permissions.rb

```diff
@@ -0,0 +1,9 @@
+class SeedNewStorytimeActionsAndPermissions < ActiveRecord::Migration
+  def up
+    Storytime::Action.seed
+    Storytime::Permission.seed
+  end
```

Comment:
Ok, let's go ahead and delete this migration then. It looks like the create_storytime_subscriptions migration already calls both of these, so it should be good to go.